$NetBSD: patch-aa,v 1.4 1999/03/05 09:20:16 agc Exp $

Install the program as gmake, the man page as gmake.1

--- configure-orig	Fri Sep 19 20:37:14 1997
+++ configure	Mon Dec  1 20:41:54 1997
@@ -59,7 +59,7 @@
 prefix=NONE
 program_prefix=NONE
 program_suffix=NONE
-program_transform_name=s,x,x,
+program_transform_name='s,^,g,'
 silent=
 site=
 srcdir=
